tap_hubspot/__init__.py

```diff
@@ -963,7 +972,7 @@ class Stream(object):
     Stream('campaigns', sync_campaigns, ["id"], None, 'FULL_TABLE'),
     Stream('contact_lists', sync_contact_lists, ["listId"], 'updatedAt', 'FULL_TABLE'),
     Stream('companies', sync_companies, ["companyId"], 'hs_lastmodifieddate', 'FULL_TABLE'),
-    Stream('deals', sync_deals, ["dealId"], 'hs_lastmodifieddate', 'FULL_TABLE'),
+    Stream('deals', sync_deals, ["dealId"], 'property_hs_lastmodifieddate', 'INCREMENTAL'),
```

Comment:
Can you please move this under https://github.com/singer-io/tap-hubspot/pull/195/files#diff-fa09b051e2cdacf5d892f6da3664b221836c73217ae4f4b8ef2b87ccbbca220dR963

Reply:
Updated